content/en/seavoice/discord/discord_bot.md

```diff
@@ -48,28 +55,39 @@ To open the voice channel chat panel, click the chat icon next to the voice chan
 <img width="60%" src="/images/discord/voice-chat-channel.png">
 </center>
 
+<p style="color:#19b6c0">Pro Tip #2:</p>
+
+If you want to temporarily stop the bot from listening to you (like pausing the session), you can right-click on the bot in the voice channel and check `Deafen Server`. This will prevent any audio data from being sent to the bot until it is un-checked. This way, you can pause the transcription and then pick your session back up when you're ready without having to stop and start a new one!
+
```

Comment:
I was thinking of having some pause feature. Is this a temporarily solution or is this what we will suggest to all users?

Reply:
I think this is just what we will suggest - I don't think we could really implement anything that would be an improvement over what Discord already offers, and it wouldn't even be a trivial feature to add, I think it would be quite complicated